test/sharness/t0125-twonode.sh

```diff
@@ -89,11 +89,6 @@ test_expect_success "set up tcp testbed" '
 	iptb init -n 2 -p 0 -f --bootstrap=none
 '
 
-# test multiplex muxer
-export LIBP2P_MUX_PREFS="/mplex/6.7.0"
-run_advanced_test "--enable-mplex-experiment"
-unset LIBP2P_MUX_PREFS
```

Comment:
go-multiplexer is very broken, has been failing travis for a while, and @whyrusleeping told me to simply remove it. I've removed the test cases in this PR and will remove the code in another (so this one can move forward).